.eslintrc.json

```diff
@@ -0,0 +1,30 @@
+{
+  "env": {
+    "browser": true,
+    "jasmine": true,
+    "jquery": true,
+    "amd": true
+  },
+  "extends": "airbnb-base/legacy",
+  "globals": {
+    "edx": true,
+    "RequireJS": true
+  },
+  "parserOptions": {
+      "ecmaVersion": 5
+  },
+  "rules": {
```

Comment:
I agree re: `avoidEscape: true`, looks like [Airbnb has that turned on](https://github.com/airbnb/javascript/blob/d45349497a712395850bd5a24b7ded15e90913a1/packages/eslint-config-airbnb-base/rules/style.js#L125) in their base config so we'll get that in here too. (Although they're using the deprecated form of expressing the rule... tsk tsk)